components/tyk-bootstrap/README.md

```diff
@@ -13,25 +13,55 @@ The chart provides little benefits by itself and should only be used in conjunct
 
 To install the chart from the Helm repository in namespace `tyk` with the release name `tyk-bootstrap`:
 
-    helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
-    helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml --devel
-    helm install tyk-bootstrap tyk-helm/tyk-bootstrap -n tyk --create-namespace -f values-bootstrap.yaml --devel
-
-
+```bash
+helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml --devel
+helm install tyk-bootstrap tyk-helm/tyk-bootstrap -n tyk --create-namespace -f values-bootstrap.yaml --devel
+```
 
 ## Uninstalling the Chart
 
-    helm uninstall tyk-bootstrap -n tyk
+```bash
+helm uninstall tyk-bootstrap -n tyk
+```
 
 This removes all the Kubernetes components associated with the chart and deletes the release.
 
 ## Upgrading Chart
 
-    helm upgrade tyk-bootstrap tyk-helm/tyk-bootstrap -n tyk
+``bash
+helm upgrade tyk-bootstrap tyk-helm/tyk-bootstrap -n tyk
+``
 
 ## Configuration
 See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To get all configurable options with detailed comments:
 
-    helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml --devel
+```bash
+helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml --devel
+```
     
 You can update any value in your local values.yaml file and use `-f [filename]` flag to override default values during installation. Alternatively, you can use `--set` flag to set it in Tyk installation.
+
+### Environment Variables
```

Comment:
Instead of defining environment variables here again should we just link it to k8s-bootstrap?

Reply:
I can 👍 we'll update accordingly

Reply:
updated in https://github.com/TykTechnologies/tyk-charts/pull/158/commits/dc72c35a650db34c2a427642d10c776b5f32e739. also, i am making this PR to draft until we have a release on tyk-k8s-bootstrap. Then, I'll update the image tags here in this PR and make the PR ready for testing. What do your think?

Reply:
Sounds good.